apps/ledger-live-mobile/src/screens/Onboarding/steps/accessExistingWallet.tsx

```diff
@@ -60,18 +77,34 @@ function AccessExistingWallet() {
             onPress: connect,
             icon: <Icons.Bluetooth color={colors.primary.c80} />,
           },
-          {
-            title: t("onboarding.welcomeBackStep.sync"),
-            event: "button_clicked",
-            eventProperties: {
-              button: "Sync with Desktop",
-            },
-            testID: "Existing Wallet | Sync",
-            onPress: sync,
-            icon: <Icons.QrCodeScanner color={colors.primary.c80} />,
-          },
+          ...(!isWalletSyncEnabled
+            ? [
+                {
+                  title: t("onboarding.welcomeBackStep.sync"),
+                  event: "button_clicked",
+                  eventProperties: {
+                    button: "Sync with Desktop",
+                  },
+                  testID: "Existing Wallet | Sync",
+                  onPress: sync,
+                  icon: <Icons.QrCodeScanner color={colors.primary.c80} />,
+                },
```

Comment:
you can do the opposite, in general it's more readable to say 

`If isWalletSyncEnabled then`  :)